java/spacewalk-java.spec

```
@@ -422,10 +425,7 @@ done
 sed -i 's/apache2.service/%{apache2}.service/' scripts/taskomatic.service
 
 %build
-PRODUCT_NAME="SUSE Manager"
-%if !0%{?sle_version} || 0%{?is_opensuse} || 0%{?rhel} || 0%{?fedora}
-PRODUCT_NAME="Uyuni"
-%endif
+PRODUCT_NAME=%{productprettyname}
```

You miss a quote here
```suggestion
PRODUCT_NAME="%{productprettyname}"
```